packages/datasource/src/api/org/checkName/index.spec.js

```diff
@@ -0,0 +1,25 @@
+import { checkName } from './index';
+import { mockCheckName } from './mock';
+import { Constants } from '../../index';
+import { jestMock } from '../../mock.jest.utils';
+
+describe('api/org/checkName', () => {
+  jestMock(mockCheckName);
+
+  it('should pass', async () => {
+    const result = await checkName({ name: 'foo_bar' });
+    expect(result.detail).toBe(Constants.SUCCESS_NOT_USED);
+    expect(result.data).toBe(undefined);
+  });
+
+  it('should failed', async (done) => {
+    try {
+      await checkName({ name: 'foo' });
+      done.fail('should not passed');
+    } catch (e) {
+      expect(e.detail).toBe(Constants.ERR_PARAMS_WRONG);
+      expect(e.errors.name[0]).toBe("It's already been used");
+      done();
+    }
+  });
+});
```

Comment:
The method of testing the mock APIs is awesome. But when it migrates to the independent mock server, these mock may be expired. We could consider adding them back later, but it's not a priority.

Reply:
I have removed all mocking and testing, for they are not useful now.